apps/editor.planx.uk/.storybook/main.ts

```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  stories: ["../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  stories: [
+    "../src/**/*.stories.@(js|jsx|ts|tsx)",
+    "../src/@planx/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
```

Comment:
These two globs should be identical, but I need to add this to get the other stories to show 🤷‍♂️